app/serializers/component_serializer/heading1_component_serializer.rb

```diff
@@ -15,16 +16,23 @@ class Heading1ComponentSerializer < ComponentSerializer::BaseComponentSerializer
     #  backend_data_for_translation = 'beta.parliament.uk'
     #  boolean = true
     #  ComponentSerializer::Heading1ComponentSerializer.new(subheading_content: string_or_translation_key, subheading_data: {"link": backend_data_for_translation}, heading_content: string_or_translation_key, heading_data: {"link": backend_data_for_translation}, context_content: string_or_translation_key, context_data: {"link": backend_data_for_translation}, context_hidden: boolean).to_h
-    def initialize(subheading_content: nil, subheading_data: nil, heading_content: nil, heading_data: nil, context_content: nil, context_data: nil, context_hidden: nil)
+    def initialize(subheading_content: nil, subheading_data: nil, subheading_link: nil, heading_content: nil, heading_data: nil, context_content: nil, context_data: nil, context_hidden: nil)
       @subheading_content = subheading_content
       @subheading_data = subheading_data
+      @subheading_link = subheading_link
       @heading_content = heading_content
       @heading_data = heading_data
       @context_content = context_content
       @context_data = context_data
       @context_hidden = context_hidden
     end
 
+    def to_s
+      return "#{@subheading_content} - #{@heading_content}" unless @subheading_content.nil?
```

Comment:
What happens if there is no `@heading_content` value?

Reply:
I'm not sure about this. I feel like at minimum it makes sense for an instance of `Heading1ComponentSerializer` to have a `@heading_content`. If not, the worst thing that would happen is this method would return `nil`.

Is there maybe something better we can do here?

Reply:
Elsewhere we've got the string `[XYZ Unavailable]`, not sure if that is appropriate here?

Reply:
So in `GroupsShowPageSerializer` there is:
```ruby
def meta
  super(title: title)
end

def title
  @group.try(:groupName) || t('no_name')
end
```
Should I amend `#to_s` to be something like:

```ruby
def to_s
  return "#{@subheading_content} - #{@heading_content}" unless @subheading_content.nil?

  @heading_content || "[Name Unavailable]"
end
```

Reply:
Yep, I'd just go with `@heading_content || t('no_name')`